pkg/network/protocols/http/telemetry.go

```diff
@@ -11,15 +11,18 @@ package http
 import (
 	"time"
 
-	"go.uber.org/atomic"
-
 	libtelemetry "github.com/DataDog/datadog-agent/pkg/network/protocols/telemetry"
+	nettelemetry "github.com/DataDog/datadog-agent/pkg/network/telemetry"
 	"github.com/DataDog/datadog-agent/pkg/util/log"
 )
 
-type telemetry struct {
-	then *atomic.Int64
+var httpTelemetry = struct {
+	lastCheck nettelemetry.StatGaugeWrapper
+}{
+	nettelemetry.NewStatGaugeWrapper("usm", "last_check", []string{}, "Gauge tracking the timestamp of when gauges were last updated"),
```

Comment:
Is telemetry of an absolute time value useful? I don't think this was reported before, but just used for the agent status check.

Reply:
reverted back to atomic here and in the other use case